admin_guide/scheduling/pod_affinity.adoc

```diff
@@ -0,0 +1,364 @@
+      requiredDuringSchedulingIgnoredDuringExecution:
+      - labelSelector:
+          matchExpressions:
+          - key: security
+            operator: In
+            values:
+            - S1
+        topologyKey: failure-domain.beta.kubernetes.io/zone
+----
++
+.. Specify an `operator`. The xref:../../rest_api/openshift_v1.adoc#unversioned-labelselectorrequirement[operator] can be `In`, `NotIn`, `Exists`, or `DoesNotExist`. For example, use the operator `In` to require the label to be in the node.
++
+.. Specify a `topologyKey`, which is a prepopulated link:https://kubernetes.io/docs/concepts/configuration/assign-pod-node/#interlude-built-in-node-labels[Kubernetes label] that the system uses to denote such a topology domain.
+
+. Create the pod.
++
+----
+$ oc create -f <pod-spec>.yaml
+----
+
+[[admin-guide-sched-affinity-config-pod-pref]]
+=== Configuring an Anti-affinity Rule
+
+The following steps demonstrate a simple two-pod configuration that creates pod with a label and a pod that uses an anti-affinity preferred rule to attempt to prevent scheduling with that pod.
+
+. Create a pod with a specific label in the pod specification:
++
+----
+$ cat team4.yaml
+apiVersion: v1
+kind: Pod
+metadata:
+  name: security-s2
+  labels:
+    security: S2
+spec:
+  containers:
+  - name: security-s2
+    image: docker.io/ocpqe/hello-pod
+----
+
+. When creating other pods, edit the pod specification to set the following parameters:
+
+. Use the `podAffinity` stanza to configure the `requiredDuringSchedulingIgnoredDuringExecution` parameter or `preferredDuringSchedulingIgnoredDuringExecution` parameter:
++
+.. Specify a weight for the node, 1-100. The node that with highest weight is preferred.
++
+.. Specify the key and values that must be met. If you want the new pod to not be scheduled with the other pod, use the same `key` and `value` parameters as the label on the first pod.
++
+----
+[source,yaml]
+    podAntiAffinity:
+      preferredDuringSchedulingIgnoredDuringExecution:
+      - weight: 100
+        podAffinityTerm:
+          labelSelector:
+            matchExpressions:
+            - key: security
+              operator: In
+              values:
+              - S2
+          topologyKey: kubernetes.io/hostname
+----
++
+.. For a preferred rule, specify a weight, 1-100.
++
+.. Specify an `operator`. The xref:../../rest_api/openshift_v1.adoc#unversioned-labelselectorrequirement[operator] can be `In`, `NotIn`, `Exists`, or `DoesNotExist`. For example, use the operator `In` to require the label to be in the node.
+
+. Specify a `topologyKey`, which is a prepopulated link:https://kubernetes.io/docs/concepts/configuration/assign-pod-node/#interlude-built-in-node-labels[Kubernetes label] that the system uses to denote such a topology domain.
+
+. Create the pod.
++
+----
+$ oc create -f <pod-spec>.yaml
+----
+
+[[admin-guide-sched-affinity-examples]]
+== Examples
+
+The following examples demonstrate pod affinity and pod anti-affinity.
```

Comment:
This might need more or different explanation. If we are only now seeing examples of pod affinity and anti-affinity, what did the examples before this show?